src/server/services/procedures/mauna-loa-co2-data/mauna-loa-co2-data.js

```diff
@@ -7,24 +7,20 @@
  * @category Science
  * @category Climate
  */
+
+const fs = require('fs');
+const path = require('path');
+
 const data = (function() {
-    const fs = require('fs');
-    const path = require('path');
     const filename = path.join(__dirname,'co2_mm_mlo.txt');
     const lines = fs.readFileSync(filename, 'utf8').split('\n');
-
-    while (lines[0].startsWith('#')) {
-        lines.shift();
-    }
-    return lines.map(line => {
-        const [,, date, avg, interpolated, trend] = line.split(/\s+/)
-            .map(txt => parseFloat(txt));
-        return {date, avg, interpolated, trend};
+    return lines.filter(x => !x.startsWith('#')).map(line => {
+        const [,, date, interpolated, trend] = line.split(/\s+/).map(parseFloat);
```

Comment:
It's not obvious to me that the "average" values in the new dataset are the same as the "interpolated" (as opposed to the "average" column) in the old dataset.

I think this is fine but wanted to make a note here in case there are any surprises after updating as this is a possible culprit.

Reply:
On line 49 of the data file it comments that missing months are interpolated. The 'average' column in the old data set occasionally had `-99.99` placeholders. So I think they just got rid of the 'average' column and replaced it with 'interpolated'.